File: API/DormManagementApi/Models/AccommodationSessionDto.cs
namespace DormManagementApi.Models
{
    public class AccommodationSessionDto
    {
        public int Id { get; set; }
        public string Name { get; set; }
        public int Active { get; set; }
        public DateOnly ApplicationPhaseStartDate { get; set; }
        public DateOnly ApplicationPhaseEndDate { get; set; }
        public DateOnly AssignmentPhaseStartDate { get; set; }
        public DateOnly AssignmentPhaseEndDate { get; set; }
        public DateOnly ReassignmentPhaseStartDate { get; set; }
        public DateOnly ReassignmentPhaseEndDate { get; set; }


    }
}

File: WEB/DormAccommodation/src/app/models/accommodation.session.dto.ts
export class AccommodationSessionDto {

  id: number;
  name: string;
  active: number;
  applicationPhaseStartDate: string;
  applicationPhaseEndDate: string;
  assignmentPhaseStartDate: string;
  assignmentPhaseEndDate: string;
  reassignmentPhaseStartDate: string;
  reassignmentPhaseEndDate: string;


  constructor(id: number, name: string,active: number,
              applicationPeriodStart: string, applicationPeriodEnd: string,
              assignmentPeriodStart: string, assignmentPeriodEnd: string,
              reassignmentPeriodStart: string, reassignmentPeriodEnd: string) {

    this.id = id;
    this.name = name;
    this.active = 1;
    this.applicationPhaseStartDate = applicationPeriodStart;
    this.applicationPhaseEndDate = applicationPeriodEnd;
    this.assignmentPhaseStartDate = assignmentPeriodStart;
    this.assignmentPhaseEndDate = assignmentPeriodEnd;
    this.reassignmentPhaseStartDate = reassignmentPeriodStart;
    this.reassignmentPhaseEndDate = reassignmentPeriodEnd;
  }


}
